README.rst

```diff
@@ -87,6 +87,18 @@ conftest.py file:
               extra.append(pytest_html.extras.html('<div>Additional HTML</div>'))
           report.extra = extra
 
+Creating a single file to the project
+-------------------------------------
+
+In order to respect the `Content Security Policy (CSP) <https://developer.mozilla.org/docs/Web/Security/CSP>`_, the CSS file is created apart by default. But if it is necessary to you create a single file to the whole report, you may run the file by the following way:
```

Comment:
As we know there are more files going to be split out, let's not single out the CSS file in the documentation. We could say that 'several assets such as CSS and images' are stored separately by default'.